<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDetailPage</class>
 <widget class="QWidget" name="PatientDetailPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1198</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>患者信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="caseRecordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>病例详情:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="caseRecord">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>388</width>
     <height>271</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;患者病例记录..&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="adviceLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>医嘱:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="adviceEdit">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>180</y>
     <width>351</width>
     <height>281</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;医嘱内容...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prescriptionBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>留下医嘱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>106</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
